packages/ckeditor5-dev-env/lib/utils/changelog.js

```diff
@@ -0,0 +1,249 @@
+/**
```

Comment:
Let's not create such a big utils files. Split it into smaller ones, preferably a function per file (but sometimes it makes a bit more sense to group them). And do this after https://github.com/ckeditor/ckeditor5-dev/pull/8/files#r97547555, because then you'll see what's the best way to shape the code.